Encounter Me/Pages/TrailProgress.razor

```diff
@@ -195,7 +190,8 @@
         ModalControl.OpenModal(_successModal);
         //used to update UI elements
         StateHasChanged();
-        int xpGain = LevelAndXp.XpFromTrail(_trailInProgress);
+        //int xpGain = LevelAndXp.XpFromTrail(_trailInProgress);
+        int xpGain = _capturePointInProgress.DefenseLevel * 50 + 100;
```

Comment:
Just my opinion, but I think that `xpGain` logic may change in the future, and then it could be hard to find where to change these hard-coded parameters. I would suggest to introduce them as fields  ( `multiplicateBy` and `addValue`) for this class. Of course, one may argue that it's unnecessary to create fields only for that and then have them hanging amongst other props and fields.